projects/storefrontlib/src/cms-components/myaccount/my-coupons/coupon-card/coupon-card.component.spec.ts

```diff
@@ -39,6 +40,7 @@ describe('CouponCardComponent', () => {
 
   beforeEach(() => {
     fixture = TestBed.createComponent(CouponCardComponent);
+    el = fixture.debugElement;
     component = fixture.componentInstance;
     component.coupon = mockCoupon;
     component.couponLoading = false;
```

Comment:
input/output - can we using a test host to simulate the real usage https://angular.io/guide/testing#component-inside-a-test-host